frontend/finance-tracker/src/components/auth/login-form.tsx

```
@@ -0,0 +1,185 @@
+                                                    href="#"
+                                                    className="ml-auto inline-block text-sm underline"
+                                                >
+                                                    Forgot your password?
+                                                </Link>
+                                            </div>
+                                            <FormControl>
+                                                <PasswordInput
+                                                    id="password"
+                                                    placeholder="******"
+                                                    autoComplete="current-password"
+                                                    tabIndex={2}
```

Biome (1.9.4)

[error] Avoid positive values for the tabIndex prop.

Elements with a positive tabIndex override natural page content order. This causes elements without a positive tab index to come last when navigating using a keyboard.
Use only 0 and -1 as tabIndex values. Avoid using tabIndex values greater than 0 and CSS properties that can change the order of focusable HTML elements.
Unsafe fix: Replace the tableIndex prop value with 0.


(lint/a11y/noPositiveTabindex)